datalad/interface/tests/test_run.py

```diff
@@ -405,7 +405,7 @@ def test_rerun_subdir(path):
     subdir = opj(path, 'subdir')
     mkdir(subdir)
     with chpwd(subdir):
-        run("python -c 'open(\"test.dat\", \"wb\").close()'")
+        run("touch test.dat")
```

Comment:
although we do not yet (fully) support windows, we would come back to it eventually -- would there be `touch`?
if it was just to fix "python" - `sys.executable` could be used

Reply:
Yeah, I suppose that's a larger issue for `test_run` because it uses `touch` elsewhere (and `echo`, `cat`, maybe others)

Reply:
ok then